scheduler/src/cook/kubernetes/api.clj

```diff
@@ -759,7 +777,9 @@
     ;; Note that node-unschedulable may be nil or false or true.
     :let [node-unschedulable (some-> node .getSpec .getUnschedulable)]
     node-unschedulable (do
-                         (log/info "Filtering out" node-name "because it is unschedulable" node-unschedulable)
+                         (log-structured/info "Filtering out node because it is unschedulable"
```

Comment:
I think this change is still worth making so that these messages are more readable when looking at the logs. I'll submit another MR for the aggregate counters.